frontend/test/pytest/test_mid_circuit_measurement.py

```diff
@@ -218,15 +222,185 @@ def circuit(x):
         assert jnp.allclose(circuit(0.0), 0)
         assert jnp.allclose(circuit(jnp.pi), 1)
 
+    def test_mcm_method_deferred_error(self, backend):
+        """Test that an error is raised if trying to execute with mcm_method="deferred"."""
+        dev = qml.device(backend, wires=1)
+
+        @qjit
+        @qml.qnode(dev, mcm_method="deferred")
+        def circuit(_x):  # pylint: disable=unused-argument
+            # QNode has unused argument because if there are no arguments then compilation happens
+            # during QJIT.__init__ rather than __call__
+            m = measure(0)
```

Comment:
This is unrelated to the above right?
```suggestion
            _ = measure(0)
```